src/main/java/org/threeten/extra/PackedFields.java

```diff
@@ -115,7 +115,7 @@ private PackedFields() {
     /**
      * Implementation of packed date.
      */
-    private static enum PackedDate implements TemporalField {
+    private enum PackedDate implements TemporalField {
```

Comment:
I take the view that `static` modifiers like this should be explicit, rather than forcing developers to remember all the compiler/spec rules.

Reply:
I've re-added the static modifiers in commit 15849c2

For reference please see [Oracle docs](http://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.9)

Reply:
Thanks. BTW I'm aware that `static` is implicit, but this is about code readability. See [here](https://checkstyle.sourceforge.io/config_modifier.html#ClassMemberImpliedModifier) and [here](https://blog.joda.org/2016/09/private-methods-in-interfaces-in-java-9.html) for more info on the rationale behind my choice.